### ISSUE TITLE
Create a template for CNN Transfer Learning and simple classification

### ISSUE THREAD

**Issue by rojberr** [GHOST-Science-Club/tree-classification-irim#9]
### Intro

The goal of this task is to create a Google Colab template for a "warm-up" classification using two tree species.

Framework of choice: PyTorch Lightning (PTL).

Please put the template in this [Colab](https://colab.research.google.com/drive/16kqlQGf8eVPNYRUS8_zB623S_XO4e2Al?usp=sharing).

Useful [tutorial](https://lightning.ai/lightning-ai/studios/finetune-an-image-classifier-with-hugging-face).

### Steps to follow:

1. Load data
Dataset: [Pure forest HF](https://huggingface.co/datasets/IGNF/PureForest/tree/main/data)
Load only two subsets of this dataset corresponding to:
    - `chestnut` (https://huggingface.co/datasets/IGNF/PureForest/blob/main/data/imagery-Castanea_sativa.zip)
    - `black pine` (https://huggingface.co/datasets/IGNF/PureForest/blob/main/data/imagery-Pinus_nigra.zip).
Useful resource: [Loading a subset of HF dataset](https://huggingface.co/docs/datasets/en/loading#hugging-face-hub)
2. Downsample the data for a uniform distribution of samples across the two classes.
3. Load a pre-trained model of choice. I suggest a small ResNet.
4. Split into train and test splits (if not provided in the dataset).
5. Create train and test `data loaders`.
6. Define a `Trainer`.
7. Train the model.
8. Report accuracy of the pre-trained model on both train and test sets.

### Remarks

- use GPU provided by Colab to speed up training

### Additional resources:
[Image Classification with PyTorch Lightning](https://lightning.ai/lightning-ai/studios/image-classification-with-pytorch-lightning)
[Loading a pre-trained ResNet in PTL](https://lightning.ai/docs/pytorch/stable/advanced/transfer_learning.html#example-imagenet-computer-vision)
